### PR TITLE
Was reading and had a few more minor copy edits

### PR DESCRIPTION
**Pull request by zsusswein**

### CODE REVIEW

File: epinow2.Rmd [hunk 1]
```diff
@@ -105,16 +110,16 @@ covid_hospitalizations <- covidcast_signal(
...
-But when evaluating methods, we're often interested in how they would have performed
+When evaluating methods, we're often interested in how they would have performed
 on the data available at the time rather than the data in its final, fully-reported state.
 This data format is equivalent to having a linelist with columns for the date of hospitalization and the date of report.
 
 > &#128161; We use the terms **report date** and **vintage** to indicate the as-of date (on what date was this snapshot of the data observed). We use **event date** or simply **date** to indicate where a data point falls along the x-axis of the epidemic time series. 
 
-> &#9888; **Don't run this command yourself if you don't have a [`covidcast` API key](https://cmu-delphi.github.io/covidcast/covidcastR/reference/covidcast_signal.html) key loaded**!. Instead go to the next step.
+If you're interested in downloading the vintages from winter 2021-2022 yourself, 
+heck out the hidden code chunk below.
```
Comment (seabbs) [inline epinow2.Rmd:120]: heck -> check again linebreaks

**Review by seabbs — CHANGES_REQUESTED** [review]
Very nice - just one typo and a query about linebreaks. Could you rerender the tutorial before we merge so we don't have to do that once its on main (though we could just add a github action to do this..)

File: epinow2.Rmd [hunk 2]
```diff
@@ -85,10 +85,15 @@ in `epinow2.Rmd`.
 
 We use the [covidcast](https://cmu-delphi.github.io/covidcast/covidcastR/) package to read in [HHS data](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/hhs.html) on confirmed Covid-19 hospital admissions. 
 The primary source for these data is the HHS state-level COVID-19 hospitalization time series, which can be found on [HealthData.gov](https://healthdata.gov/Hospital/COVID-19-Reported-Patient-Impact-and-Hospital-Capa/g62h-syeh).
-The [covidcast](https://cmu-delphi.github.io/covidcast/covidcastR/) package provides a convenient way to read in a subset of the full data set.
+The [covidcast](https://cmu-delphi.github.io/covidcast/covidcastR/) package provides a convenient way to read in a subset of the full dataset.
 Here, we focus on the data from New York, Utah, Ohio, Virginia, and North Carolina during the Omicron wave in the winter of 2021-2022. 
 
-We _could_ download the most recent version of this data directly using:
+We _could_ download the most recent version of this data directly. If you're
```
[the remark this answers is not in the document]
Comment (zsusswein) [reply epinow2.Rmd:91]: These are rendered out in the html. I was keeping the rmd to <80 chars a line, but happy to do longer if that's your convention.

**Comment by seabbs** [comment]
Fair enough on the linebreaks. I don't typically line break rmd so would rather not. 

Definitely would rather have it be inconsistent so one way or other would be ideal for all the text.

**Comment by zsusswein** [comment]
Cool swapped  'em out for longer lines

**Comment by seabbs** [comment]
Haha consistent not inconsistent. That would be worrying if it was my desire.

**Review by seabbs — APPROVED** [review]
LGTM
haha you spotted I didn't know what the data was called.

**Comment by kgostic** [comment]
Approved!

**Pull request merged**